src/main/java/service/operation/ReturnOperation.java

```diff
@@ -0,0 +1,16 @@
+package service.operation;
+
+import dao.FruitDao;
+
+public class ReturnOperation implements OperationHandler {
+    private final FruitDao fruitDao;
+
+    public ReturnOperation(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public void handle(String fruit, int quantity) {
+        fruitDao.updateBalance(fruit,fruitDao.getBalance(fruit) + quantity);
```

Comment:
```suggestion
        fruitDao.updateBalance(fruit, fruitDao.getBalance(fruit) + quantity);
```
don't forget to Ctrl Alt L throughout your files